### DISCUSSION TITLE
Can I use these fonts in a personal project?

### DISCUSSION THREAD

**Discussion by tusqasi** [ryanoasis/nerd-fonts#768] [Q&A] [answered]
I wanted to use the blex mono font in a project on mine, usually I would have found the license and included that. But I can't find anything for blex mono.

**Comment by Finii** [comment, marked as answer]
https://github.com/ryanoasis/nerd-fonts/blob/master/patched-fonts/IBMPlexMono/Mono/complete/LICENSE.txt

**Reply by tusqasi** [reply]
Thanks. I didn't look all the way in there.
